### PR TITLE
Update docs for hive.write-precision to include writes

### PR DESCRIPTION
**Pull request by jirassimok**
This replaces "write opreations are only supported for `MILLISECONDS`" with "values with higher precision than configured will be rounded," and adds "in Trino" and "Hive" to make the preceding sentence clearer.

### CODE REVIEW

File: docs/src/main/sphinx/connector/hive.rst [hunk 1]
```diff
@@ -294,10 +294,10 @@ Property Name                                      Description
 ``hive.orc.time-zone``                             Sets the default time zone for legacy ORC files that did     JVM default
                                                    not declare a time zone.
 
-``hive.timestamp-precision``                       Specifies the precision to use for columns of type           ``MILLISECONDS``
-                                                   ``timestamp``. Possible values are ``MILLISECONDS``,
-                                                   ``MICROSECONDS`` and ``NANOSECONDS``. Write operations
-                                                   are only supported for ``MILLISECONDS``.
+``hive.timestamp-precision``                       Specifies the precision to use in Trino for Hive columns     ``MILLISECONDS``
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/hive.rst:297]: remove redundant "in Trino"

File: docs/src/main/sphinx/connector/hive.rst [hunk 2]
```diff
@@ -294,10 +294,10 @@ Property Name                                      Description
...
+                                                   of type ``timestamp``. Possible values are ``MILLISECONDS``,
+                                                   ``MICROSECONDS`` and ``NANOSECONDS``. Values with higher
+                                                   precision than configured will be rounded.
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/hive.rst:300]: are rounded.

**Review by mosabua — CHANGES_REQUESTED** [review]
Minor tweaks only...

**Review by findepi — APPROVED** [review]

File: docs/src/main/sphinx/release/release-352.md [hunk 3]
```diff
@@ -59,8 +59,9 @@
 * Match columns by name rather than by index by default for Parquet files. ({issue}`6479`)
 * Remove the `hive.partition-use-column-names` configuration property and the
   `partition_use_column_names ` session property. This is now determined automatically. ({issue}`6479`)
-* Support reading timestamp with microsecond or nanosecond precision (as configured with 
+* Support reading timestamps with microsecond or nanosecond precision (as configured with
   `hive.timestamp-precision` property) nested within `array`, `map` or `struct` data types. ({issue}`5195`)
+* Support writing timestamps with microsecond or nanosecond precision. ({issue}`3977`)
```
Comment (jirassimok) [inline docs/src/main/sphinx/release/release-352.md:64]: I don't think there's a more-specific issue than #3977 to link here. There's more-specific issues for timestamps in arrays (#5195 and #6760), but not for writing in general.

File: docs/src/main/sphinx/release/release-352.md [hunk 4]
```diff
@@ -59,8 +59,9 @@
 * Match columns by name rather than by index by default for Parquet files. ({issue}`6479`)
 * Remove the `hive.partition-use-column-names` configuration property and the
   `partition_use_column_names ` session property. This is now determined automatically. ({issue}`6479`)
-* Support reading timestamp with microsecond or nanosecond precision (as configured with 
+* Support reading timestamps with microsecond or nanosecond precision (as configured with
```
Comment (findepi) [inline docs/src/main/sphinx/release/release-352.md:62]: I don't have a string opinion whether plural or singular, but let's keep the original wording.

File: docs/src/main/sphinx/release/release-352.md (hunk 3 above)
Comment (findepi) [reply docs/src/main/sphinx/release/release-352.md:64]: And if  we were to link to a PR or two, which ones would that be?

**Review by mosabua — APPROVED** [review]
Looks good now.

**Pull request merged**